tests/IsoCodes/Tests/CreditCardTest.php

```diff
@@ -30,7 +30,6 @@ public function getValidCreditCards()
             array('4903010000000009'),  //Switch
             array('4111111111111111'),  //Visa
             array('6304100000000008'),  //Laser
-            array(6304100000000008),    //Laser
```

Comment:
Comment not relevant anymore, test is here: https://github.com/ronanguilloux/IsoCodes/pull/93/files#diff-391157587170daf579f5855197312e0aR33